LetsDoStuff.WebApi/Services/DTO/AcceptRejectRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LetsDoStuff.WebApi.Services.DTO
+{
+    public class AcceptRejectRequest
+    {
+        public int ActivityId { get; set; }
+
+        public int ParticipanteId { get; set; }
+    }
```

Comment:
What is the Participante?

Reply:
It's a Spanish participant... Or maybe it's just my bad writing skill :)